zerver/migrations/0050_auto_20170105_0343.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
```

Comment:
This file can be renamed to something appropriate (see the other files in `zerver/migration` for examples)